h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
```

Comment:
Yeah, that's a good point.

Looking further up this file, I see this:

```python
     authority_groups = (request.find_service(name='authority_group')
                         .public_groups(authority=authority))
  
      groups = authority_groups + _user_groups(user)
```

Which to me seems to me to hint that this file is doing a bunch too much work already. Perhaps let's move the retrieval of the list of groups for this endpoint into its own service, which can optimise the query it uses for its own purposes:

```python
(session.query(models.Group)
     .options(joinedload(models.Group.creator).load_only("id"))
```

I'm guessing that would allow getting rid of the `authority_group` service?